source/aggregation-pipeline-builder.txt

```diff
@@ -172,6 +176,70 @@ your chosen view mode:
             
             |compass-short| returns your results in the document view. 
 
+   .. tab:: Stage Wizard 
+      :tabid: wizard 
+
+      .. procedure::
+         :style: normal
+         
+         .. step:: Select the :guilabel:`Stages` view.
+            
+            In the aggregation pipeline pane, ensure the 
+            :guilabel:`{} Stages` toggle switch is selected.
+
+            .. figure:: /images/compass/agg-builder-stageview-toggle-on.png
+               :figwidth: 300 px
+               :alt: Toggle on stage view mode 
+
+         .. step:: Open the :guilabel:`Stage Wizard` card. 
+            
+            To the right of the view mode toggle, click the funnel icon to open 
+            the :guilabel:`Stage Wizard` card.
+
+            .. figure:: /images/compass/agg-builder-stage-wizard.png
+               :figwidth: 300 px
+               :alt: Click the icon to the right of the view mode toggle.
+
+         .. step:: Select an aggregation pipeline stage use case.
+
+            On the :guilabel:`Stage Wizard` card, select the stage use case for 
+            the first stage of the pipeline.|compass-short| populates the the 
```

Comment:
on staging `|compass-short|` renders `|compass-short|` and not Compass as it should. Prolly needs space at the start?

Reply:
Ah thanks for that catch. Updated.